core/ir/x86/decode_table.c

```diff
@@ -7241,6 +7248,27 @@ const instr_info_t rm_extensions[][8] = {
     {OP_rdpkru, 0xee0f0171, catUncategorized, "rdpkru", eax, edx, ecx, xx, xx, mrm, x, END_LIST},
     {OP_wrpkru, 0xef0f0171, catUncategorized, "wrpkru", xx, xx, ecx, edx, eax, mrm, x, END_LIST},
   },
+  { /* rm extension 6 */
+    /* XXX i#1314: this also sets eip */
+    {OP_xabort, 0xf8c60067, catUncategorized, "xabort", eax, xx, Ib, xx, xx, mrm, x, END_LIST},
```

Comment:
Let's always give a category whenever we edit an uncategorized opcode.
I think catOther is the only option here.